yt_dlp/extractor/jditemvideo.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+import json
+import random
+import time
+
+from .common import InfoExtractor
+from ..utils import determine_ext, ExtractorError, traverse_obj
+
+
+class JdItemVideoIE(InfoExtractor):
+    _VALID_URL = r"https://.+.jd.[a-z\.]{2,9}/(?P<id>\d{6,16}).html"
+
+    IE_NAME = 'jd-video'
+    IE_DESC = 'jd-video extractor'
+    _NETRC_MACHINE = False
+
+    _JD_API_VIDEO_CALLBACK_URL = 'https://cd.jd.com/tencent/video_v3?callback=jQuery{rand}&vid={video_id}&type=1&from=1&appid=24&_={timestamp}'
```

Comment:
Since this var is only used once, moving it outside of `_real_extract` actually makes the code harder to read/understand